src/configurations.md

```diff
@@ -0,0 +1,5 @@
+# Fuel Configurations
+
+Fuel is a modular architecture and is designed to operate in multiple different configurations.
+
+![a chart showing different Fuel configurations](https://user-images.githubusercontent.com/15346823/188051935-be751b92-5402-4216-aa2d-0e409fb356fd.png)
```

Comment:
Do you know what the persistence policy is for this? I think we should host this locally in the book (e.g. a relative link to an image file in some subdirectory).

Reply:
true, will update this